# request params
sensor_msgs/Image color_image
string prompt
---
# response params
string[] labels
int32[] class_id
std_msgs/Float32MultiArray boxes
float32[] scores
sensor_msgs/Image annotated_frame
std_msgs/Float32MultiArray segmasks